app/controllers/list_responses_controller.rb

```diff
@@ -0,0 +1,30 @@
+class ListResponsesController < ApplicationController
+  def feedbacks
+    authorize :list_response, :feedbacks?
```

Comment:
The second argument to `authorize` is optional here since it already defaults to the action name with `?` appended (same for the other actions)